packages/appeals-service-api/data/lpa-list.csv

```diff
@@ -328,7 +328,7 @@
 ;E60000332;U2805;North Northamptonshire Council - Corby Area;planningservices.cbc@northnorthants.gov.uk;northnorthants.gov.uk;TRUE\n
 ;E60000332;H2835;North Northamptonshire County Wellingborough;planningappeals.bcw@northnorthants.gov.uk;northnorthants.gov.uk;TRUE\n
 322;E60000333;M9570;West Northamptonshire;;;FALSE\n
-323;E69999999;Q9999;System Test Borough Council;appealplanningdecisiontest@planninginspectorate.gov.uk;;TRUE\n
+323;E69999999;Q9999;System Test Borough Council;appealplanningdecisiontest@planninginspectorate.gov.uk;planninginspectorate.gov.uk;TRUE\n
```

Comment:
the data here is supposed to reflect the latest csv uploaded to the database - if there is an LPA roll out in future, will need to confirm the domain is present for STBC or it will disappear when the newer csv replaces this one.